src/utils/config.py

```diff
@@ -402,3 +402,68 @@ def validate_construction_config_settings(config):
                 "If running NI construction, a NI construction file path must be"
                 " provided."
             )
+
+def validate_survey_config(config: dict) -> dict:
+    """Validate the configuration settings.
+
+    Args:
+        config (dict): The configuration dictionary.
+
+    Returns:
+        dict: The validated configuration dictionary.
+
+    Raises:
+        ValueError: If PNP is set and Northern Ireland data is configured to run.
+    """
+    survey_config = config.get("survey", {})
+    if survey_config.get("survey_type") == "PNP":
+        # List of values not compatible with PNP
+        values = [
+            config.get("global", {}).get("load_ni_data", True)
+        ]
+        if any(values):
+            raise ValueError(
+                "Error: PNP is set. Northern Ireland data cannot be run. Please update "
+                "the user config."
+            )
+    return config
+
+
+def validate_pnp_config(config: dict) -> None:
+    """
+        Validate the PNP configuration settings.
+
+    Args:
+        config (dict): The configuration dictionary.
+
+    Returns:
+        dict: The validated configuration dictionary.
+
+    """
+    # Check if PNP is set
+    if config["survey"]["survey_type"] == "PNP":
+        # List of values not compatible with PNP
+        incompatible_settings = ["output_ni_full_responses",
+                                 "output_mapping_ni_qa",
+                                 "output_short_form",
+                                 "load_ni_data",
+                                 "run_ni_construction",
+                                 "output_ni_sas",
+                                 "output_intram_by_pg_gb",
+                                 "output_outlier_qa"]
+ 
+        values = [config.get("global", {}).get(setting, True)
+                  for setting in incompatible_settings]
+ 
+        if any(values):
+             print(
+                "WARNING: PNP is set. The following settings are not compatible with PNP: "
+                "output_ni_full_responses, output_mapping_ni_qa, output_short_form, load_ni_data,"
+                "run_ni_construction, output_ni_sas, output_intram_by_pg_gb and output_outlier_qa."
+                " User config will be updated to False."
```

Comment:
could you have this so it only outputs the items they actually chose? Do you think that would be useful?

Reply:
Updated code so that it prints the outputs that it has changed :)